scarlet/initialization.py

```diff
@@ -508,7 +508,8 @@ def init_source(
         # deblending of the edge source, which for bright sources
         # may ruin an entire blend. So we reinitialize edge sources
         # to allow for shifting and return the result.
-        if source_shifting is False and hasEdgeFlux(source, edge_distance):
+        source.is_edge = hasEdgeFlux(source, edge_distance)
```

Comment:
This piece of code shouldn't actually be in scarlet at all. It doesn't work as intended if I understand it correctly, and it can/should be set upstream.

Reply:
That is true. I think that we can just remove this whole section now, since I should be able to just add the shifting parameter to the source class after it is created now. I don't think that was always true.